entries/a/alltrails.com.json

```
@@ -0,0 +1,14 @@
+{
+	"AllTrails": {
+		"domain": "alltrails.com",
+		"contact": {
+			"facebook": "AllTrails",
+			"twitter": "AllTrails",
+			"email": "support@alltrails.com"
+		},
+		"categories": [
+			"health",
+			"other"
```

```suggestion
			"health"
```

As the `other` category is only for entries that don't fit in any other category, it's probably best to just use the `health` category.